pom.xml

```diff
@@ -27,7 +27,7 @@
   <parent>
     <groupId>org.xwiki.commons</groupId>
```

Comment:
You should use the contrib parent instead.

Reply:
The plan was to do this change in a new commit, but if the duplicate code is just added, maybe this PR is the right place to do so. However, @oanalavinia managed to do this improvement on the application to be compatible with XWiki 7.0 and using the parent pom, she will be forced to increase the XWiki version to 7.4.x, or release a new parent version for 7.0.

Reply:
> be compatible with XWiki 7.0

Is this a string requirement ? 7.4 is quite old already and it does not make much sense to use 7.0 anyway.